src/ch/index.html

```diff
@@ -8,9 +8,9 @@
     <meta name="viewport" content="width=device-width">
 
     {% for lang in available_languages %}
-    <link rel="alternate" href="/{{lang}}/" hreflang="{{lang|replace('_', '-')}}">
+    <link rel="alternate" href="../{{lang}}/index.html" hreflang="{{lang|replace('_', '-')}}">
```

Comment:
This and all below changes are to allow the site to run under subdirectories, like the mirrors doing. One example: https://s3.amazonaws.com/3z8eilzpfmrb8/fa_IR/index.html